tests/library/trace-viewer.spec.ts

```diff
@@ -246,6 +246,31 @@ test('should have network requests', async ({ showTraceViewer }) => {
   await expect(traceViewer.networkRequests).toContainText([/200GET\/frames\/script.jsapplication\/javascript/]);
 });
 
+test('should filtering a request in network tab', async ({ showTraceViewer }) => {
```

Comment:
```suggestion
test('should filter a request in network tab', async ({ showTraceViewer }) => {
```

That's awesome! Just proposed a small grammar correction!

Reply:
modified